ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/precompiles/privacy/OnChainPrivacyPrecompiledContract.java

```diff
@@ -191,7 +191,7 @@ public Bytes compute(final Bytes input, final MessageFrame messageFrame) {
     if (messageFrame.isPersistingPrivateState()) {
 
       privateWorldStateUpdater.commit();
-      disposablePrivateState.persist();
+      disposablePrivateState.persist(null);
```

Comment:
I wonder if an optional wouldn't be an idea to avoid having to call this method with null ?

Reply:
The only argument is style really.  The forest ignores it, in bonsai mode Tthe block that switches on whether or not it is null is too large for a good looking lambda.  The bulk of the null calls are in test code.  Another option is two methods, one with no args.  But in the places it's called directly with null it's a good reminder in the code "this is not a regular commit".

Reply:
>it's called directly with null it's a good reminder in the code "this is not a regular commit". That suits me . We can leave it if it provides information and if the majority of calls like this are in the test